src/__experimental__/components/radio-button/radio-button-group.spec.js

```diff
@@ -35,6 +35,10 @@ describe('RadioButtonGroup', () => {
     expect(render()).toMatchSnapshot();
   });
 
+  it('renders as expected with option isRequired', () => {
+    expect(render(TestRenderer.create, { isRequired: true })).toMatchSnapshot();
```

Comment:
Is setting `isRequired` here actually doing anything to this snapshot? I think this can be deleted if not

Reply:
Done